easybuild/easyconfigs/s/Slideflow/Slideflow-3.0.1-foss-2023a-CUDA-12.1.1.eb

```
@@ -0,0 +1,163 @@
+easyblock = 'PythonBundle'
+
+name = 'Slideflow'
+version = '3.0.1'
+versionsuffix = '-CUDA-12.1.1'
+
+homepage = 'https://slideflow.dev/'
+description = """Slideflow is a Python package that provides a unified API for
+building and testing deep learning models for histopathology, supporting both
+Tensorflow/Keras and PyTorch."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('scikit-build', '0.17.6'),
+    ('poetry', '1.7.1'),
+]
+
+dependencies = [
+    ('aiohttp', '3.8.5'),
+    ('numba', '0.58.1'),
+    ('Python', '3.11.3'),
+    ('Python-bundle-PyPI', '2023.06'),
+    ('pyvips', '2.2.3'),
+    ('Pillow', '10.0.0'),
+    ('PyTorch', '2.1.2', '-CUDA-12.1.1'),
+    ('scikit-learn', '1.3.1'),
+    ('scikit-image', '0.22.0'),
+    ('torchvision', '0.16.0', '-CUDA-12.1.1'),
+    ('tqdm', '4.66.1'),
+    ('SWIG', '4.1.1'),
+    ('SciPy-bundle', '2023.07'),
+    ('PyOpenGL', '3.1.7'),
+    ('spaCy', '3.7.4'),
+    ('h5py', '3.9.0'),
+    ('Ninja', '1.11.1'),
```

Ninja should be a build dependency as well, not a runtime dep
```suggestion
builddependencies = [
    ('CMake', '3.26.3'),
    ('Ninja', '1.11.1'),
    ('scikit-build', '0.17.6'),
    ('poetry', '1.7.1'),
]

dependencies = [
    ('aiohttp', '3.8.5'),
    ('numba', '0.58.1'),
    ('Python', '3.11.3'),
    ('Python-bundle-PyPI', '2023.06'),
    ('pyvips', '2.2.3'),
    ('Pillow', '10.0.0'),
    ('PyTorch', '2.1.2', '-CUDA-12.1.1'),
    ('scikit-learn', '1.3.1'),
    ('scikit-image', '0.22.0'),
    ('torchvision', '0.16.0', '-CUDA-12.1.1'),
    ('tqdm', '4.66.1'),
    ('SWIG', '4.1.1'),
    ('SciPy-bundle', '2023.07'),
    ('PyOpenGL', '3.1.7'),
    ('spaCy', '3.7.4'),
    ('h5py', '3.9.0'),
```

---

Ninja was a runtime requirement for part of the package when I first made the easyconfig, the maintainer of the package has since updated it on my request, so it should be fine, yes.